src/main/java/ch/admin/bag/covidcertificate/client/signing/SigningInformationDto.java

```diff
@@ -9,84 +11,17 @@
 
 @Getter
 @EqualsAndHashCode
+@AllArgsConstructor
 @ToString
+@Builder
 public class SigningInformationDto {
-    private String certificateType;
-    private String code;
-    private String alias;
-    private String certificateAlias;
-    private LocalDate validFrom;
-    private LocalDate validTo;
+    private final String certificateType;
+    private final String code;
+    private final String alias;
+    private final String certificateAlias;
+    private final Integer slotNumber;
+    private final LocalDate validFrom;
+    private final LocalDate validTo;
     @Setter
     private String calculatedKeyIdentifier;
```

Comment:
The calculatedKeyIdentifier is a transient attribute calculated after construction of the object. That was the reason of explicit constructor and builder. This solution ist also fine :-)